src/commands/georadiusbymember_ro.json

```diff
@@ -130,6 +130,55 @@
                     }
                 ]
             }
-        ]
+        ],
+        "reply_schema": {
+            "description": "Array of matched members information",
+            "anyOf": [
+                {
+                    "description": "If no WITH option is specified, array of matched members names",
+                    "type": "array",
+                    "items": {
+                        "description": "name",
+                        "type": "string"
+                    }
+                },
+                {
+                    "type": "array",
+                    "items": {
+                        "type" : "array",
+                        "prefixItems": {
+                            "description": "name",
+                            "type": "string"
+                        },
+                        "items": {
+                            "anyOf": [
```

Comment:
@sjpotter why not `oneOf`?

Reply:
btw I think there's a bug here... prefixItems talks about all the items (i.e. it only works just because one of the `items` happens to be a `string`)

Reply:
TBH i feel that this looks better:
```
{
    "anyOf": [
         {
            "description": "If no WITH* option is specified, array of matched members names",
            "type": "array",
            "items": {
                "description": "member",
                "type": "string"
            }
        },
        {
            "description": "If WITHDIST option is specified, the distance from the center as a floating point number, in the same unit specified in the radius",
            "type": "array",
            "items": [
                {
                    "description": "member",
                    "type": "string"
                },
                {
                    "description": "distance",
                    "type": "number"
                }
            ]
        },
        {
            "description": "If WITHCOORD option is specified, the coordinates as a two items x,y array (longitude,latitude)",
            "type": "array",
            "items": [
                {
                    "description": "member",
                    "type": "string"
                },
                {
                    "type": "array",
                    "items": [
                        {
                            "description": "latitude (x)",
                            "type": "number"
                        },
                        {
                            "description": "longitude (y)",
                            "type": "number"
                        }
                    ]
                }
            ]
        },
        {
            "description": "If WITHHASH option is specified, the geohash integer",
            "type": "array",
            "items": [
                {
                    "description": "member",
                    "type": "string"
                },
                {
                    "description": "geohash",
                    "type": "string"
                }
            ]
        }
    ]
}
```

even if it's longer

Reply:
i think it may be easier for client writers as well (i.e. try to use oneOf and anyOf as far up the tree as we can)